README.md

```diff
@@ -0,0 +1,74 @@
+<h1 align="center" style="margin:1em;">
+  <a href="https://iris-grib.readthedocs.io/en/latest/">iris-grib</a>
+</h1>
+
+<h4 align="center">
+GRIB interface for <a href="https://github.com/SciTools/iris">Iris</a>.
+</h4>
+
+<p align="center">
```

Comment:
A `p` tag is a paragraph tag, and should contain a paragraph's worth of text. If you wish to wrap all the shields below in a single html element to centre-align them you should be using a `div` tag.